ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__ota-multi-stable-4.19-upgrade-from-stable-4.19.yaml

```diff
@@ -0,0 +1,82 @@
+base_images:
+  ansible:
+    name: "4.19"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.19"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: openstack-installer
+  tools:
+    name: "4.19"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  intermediate1:
+    release:
+      architecture: multi
+      channel: candidate
+      version: 4.19.0-ec.1
+  latest:
+    release:
+      architecture: multi
+      channel: candidate
+      version: 4.19.0-ec.0
+  target:
+    release:
+      architecture: multi
+      channel: candidate
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: baremetal-ipi-retarget-f28
```

Comment:
is z-then-y job needed to be enabled?

Reply:
I tried y-then-y, but 4.20 image will cause all jobs in a config file broken.